gatsby_fp/src/styles/facultyprofile.css

```diff
@@ -1,16 +1,49 @@
 /* Styles for Faculty Profile */
-
 body {
-  font-size: 1.1em;
+  font-family: "Chaparral W01", Georgia, Times, "Times New Roman", serif;
+  font-style: normal;
+  font-weight: 300;
+  min-width: 300px;
+  overflow-x: hidden;
+  overflow-wrap: break-word;
+}
+
+.img_wrapper {
+  width: 13rem;
+  margin-right: 1.5em;
 }
 
 .sectiontitle {
-  font-size: 1.4em;
-  font-weight: bold;
-  padding-top: 0.4em;
-  margin-bottom:-0.2em;
+  font-family: "Trade Gothic W01", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.875rem;
+  color: #666666;
+  padding-top: .4em;
+  margin: 24px 0 12px;
+  text-transform: none;
+}
+.bioimage{
+  margin-right: 1.5em;
```

Comment:
We don't need `margin-right` here, since we also apply it in the single-column-mode media query. Or, you could remove it from the single-column-mode media query and let this "always applied" `margin-right` happen at all times.